python/samples/concepts/memory/azure_ai_search_hotel_samples/step_2_use_as_a_plugin.py

```diff
@@ -139,7 +139,14 @@ def update_options_search(
                     type="str",
                     is_required=True,
                     type_object=str,
-                )
+                ),
+                KernelParameterMetadata(
```

Comment:
Added this because in the response, it wasn't very useful to read some random ASCII chars as the included hotel_id.